app/views/groups/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h2 class='text-center text-lg font-bold my-2'>
+  <%= @group.name %>
+</h2>
+
+<div class='bg-white p-2 m-3 text-center rounded-sm'>
+  <p class='font-bold'>Total Amount: $<%= @total_amount %></p>
+</div>
+
+<div id="payments" class='flex flex-col gap-3 m-3'>
+  <% @payments.each do |payment| %>
+    <%= render "payments/payment_card", payment: payment %>
+  <% end %>
+</div>
+
+<div class='flex justify-center'><%= link_to "New payment for this group",
```

Comment:
- [ ] Please make sure this button looks similar to that as seen in the original design.
### Your Implementation
![image](https://github.com/limavfabio/easy_pay/assets/45460192/6611747f-a9f8-4396-923e-85c8dce3a9f6)
### Required
![image](https://github.com/limavfabio/easy_pay/assets/45460192/9ec09446-5092-4105-b9fc-c879fbf762e0)